doc/Learn/Tutorials/Navigation/Advanced/HowTo-AdvancedPageNavigation.md

```diff
@@ -37,7 +36,7 @@ Sometimes when you navigate you don't want to leave the current page in the back
 
 If you run the application and navigate to the `SecondPage` the back button in the `NavigationBar` isn't visible, since the frame back-stack is empty.
 
-### 2. Navigating to a Page and Removing a Page from Back Stack
+### Navigating to a Page and Removing a Page from Back Stack
 
 Another common scenario is to navigate to a page and then remove the current page from the back stack.
 
```

Comment:
Again, we should add another example which uses XAML attribute `Navigation.Request="-Second"`